command/REPLCommand.java

```diff
@@ -379,16 +387,56 @@ public User.Create asUserCreate() {
             }
         }
 
-        public static class Password extends REPLCommand.User {
+        public static class PasswordUpdate extends REPLCommand.User {
 
-            public static String token = "password";
+            public static String token = "password-update";
             private static String helpCommand = User.token + " " + token + " " + "<username>";
             private static String description = "Update the password of user with name <username>";
 
             private final String user;
+            private final String oldPassword;
+            private final String newPassword;
+
+            public PasswordUpdate(String user, String oldPassword, String newPassword) {
+                this.user = user;
+                this.oldPassword = oldPassword;
+                this.newPassword = newPassword;
+            }
+
+            public String user() {
+                return user;
+            }
+
+            public String oldPassword() {
+                return oldPassword;
+            }
+
+            public String newPassword() {
+                return newPassword;
+            }
+
+            @Override
+            public boolean isUserPasswordUpdate() {
+                return true;
+            }
+
+            @Override
+            public PasswordUpdate asUserPasswordUpdate() {
+                return this;
+            }
+        }
+
+        public static class PasswordSet extends REPLCommand.User {
+
+            public static String token = "password-set";
```

Comment:
Console's architecture currently only supports a single layer of abstract classes, and demands that every token not have any spaces. Thus, achieving `user password set <username>` or `user password update <username>` would require either poor code or architecture changes.

Reply:
okay, please add as an issue

Reply:
https://github.com/vaticle/typedb-console/issues/198